test/support/acceptance.go

```diff
@@ -27,22 +28,24 @@ func ParseSnapshotImages() (SnapshotMap, error) {
 	return snapshotImages, nil
 }
 
-func ParseOperatorImages(helpContent string) OperatorMap {
+func ParseOperatorImages(helpContent string) (OperatorMap, OperatorMap) {
 	const minimumValidMatches = 3
 	re := regexp.MustCompile(`-([\w-]+image)\s+string[^"]+default "([^"]+)"`)
 	matches := re.FindAllStringSubmatch(helpContent, -1)
-	images := make(OperatorMap)
+	operatorSnapshotImages := make(OperatorMap)
```

Comment:
Suggestion: Rename `operatorSnapshotImages` to `operatorTasImages` for consistency with the rest of the code.